### PR TITLE
FI-941 Add us core gaps demonstrations.

### PR DESCRIPTION
**Pull request by arscan**
Addresses #151 

Adds two new visual inspection / demonstrations for the following gaps:

* Patient Demographics - Suffix
* Patient Demographics - Previous Name
* AllergyIntolerance.reaction

These will likely be removed when US Core releases an upcoming errata release.  To view these, in the Program Edition, go to 'Other' tests and run the 'Visual Inspection and Attestation' test (the very last one).

Screen Shot 2020-07-27 at 11 31 11 AM

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR: FI-941
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- n/a Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/modules/onc_program_procedure.yml [hunk 1]
```diff
@@ -1109,7 +1112,10 @@ procedure:
           not require a specific set of data to be loaded.  Because of this,
           omissions are not possible to detect.  However, the tester can use the
           tool to compare responses to what is displayed in the system, and
-          register this as an attestation.
+          register this as an attestation.  Additionally, US Core v3.1 does
+          not include two required USCDI v1 data elements for Patient Demographics
```
Comment (Jammjammjamm) [inline lib/modules/onc_program_procedure.yml:1116]: Isn't it three elements?

Comment (arscan) [inline lib/modules/onc_program_procedure.yml:1116]:
```suggestion
          not include three required USCDI v1 data elements for Patient Demographics
```

Comment (arscan) [reply lib/modules/onc_program_procedure.yml]: Yeah, there are two Patient elements and one Allergy element.  Good catch, thanks.  Fixed.
